### PR TITLE
Add Bitwise Op. from Rosettacode

### PR DESCRIPTION
**Pull request by Herve-M**

### CODE REVIEW

**Comment by privat** [comment]
add test files: `tests/rosetta_bitwise.inputs` and `tests/sav/rosetta_bitwise.res`

File: examples/rosettacode/bitwise_operations.nit [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Bitwise operations
+# SEE: <http://rosettacode.org/wiki/Bitwise_operations>
+# Without logical right shift
+
```
Comment (privat) [inline examples/rosettacode/bitwise_operations.nit]: remove this blank line

**Comment by Herve-M** [comment]
All done.

**Comment by privat** [comment]
great, just squash the commits into a single one and you will have my +1

**Pull request closed without merging**